### DISCUSSION TITLE
rdk_doc/en/RDK

### DISCUSSION THREAD

**Discussion by giscus** [D-Robotics/rdk_doc#23] [General]
# rdk_doc/en/RDK

This document is the user manual for the D-Robotics RDK Suite, providing developers with instructions and development guidance for products such as RDK X3, RDK X3 Module, and RDK Ultra. The content covers various aspects such as hardware design, system customization, application development, and algorithm toolchains. Users are welcome to update and experience, please refer to the System Burning section for specific instructions.

http://en.developer.d-robotics.cc/rdk_doc/en/RDK

**Comment by baqwas** [comment]
Is there a STEP file of the board for public access? Thanks.

**Reply by xuguosheng110** [reply]
yes you can download it from here:
https://developer.d-robotics.cc/rdk_doc/en/Advanced_development/hardware_development/rdk_x3/hardware

**Reply by baqwas** [reply]
Really appreciate the assist. Thanks!
